<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SParamDialog</class>
 <widget class="QDialog" name="SParamDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System parameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>263</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="bn_Ok">
     <property name="text">
      <string>Ok</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabPlc">
      <attribute name="title">
       <string>Контролери</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabTag">
      <attribute name="title">
       <string>Теги</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Звіти</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="writeRep">
         <property name="text">
          <string>Писати звіти</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bn_Ok</sender>
   <signal>clicked()</signal>
   <receiver>SParamDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>674</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
